pkg/features/features.go

```diff
@@ -79,6 +79,13 @@ const (
 	// alpha: v0.9.0
 	// Setting on hub side.
 	FeedInventory featuregate.Feature = "FeedInventory"
+
+	// EnableMetric enable the collection of podMetric and nodeMetric from metric server
+	//
+	// owner: @dixudx
```

Comment:
why owner is dixudx rather than you?

Reply:
Please use flag instead of feature gate.

Reply:
> Please use flag instead of feature gate.

ok, will change to flag.

Reply:
> why owner is dixudx rather than you?

ok, will change to myself

Reply:
> Please use flag instead of feature gate.

fixed